### PR TITLE
Feature/pwn 5000 default invest screen

### PR DESCRIPTION
**Pull request by bigearsenal**
## Description of the changes
-

### CODE REVIEW

File: p2p_wallet/Resources/Assets.xcassets/solend-tutorial-2.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
```
Comment (qwerkly) [inline p2p_wallet/Resources/Assets.xcassets/solend-tutorial-2.imageset/Contents.json:2]: remove png and use pdf

File: p2p_wallet/Resources/Assets.xcassets/solend-tutorial-3.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "Frame 1523523.png",
+      "idiom" : "universal",
```
Comment (qwerkly) [inline p2p_wallet/Resources/Assets.xcassets/solend-tutorial-3.imageset/Contents.json:5]: remove png and use pdf

File: p2p_wallet/Resources/Colors.xcassets/e0dbff.colorset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
```
Comment (qwerkly) [inline p2p_wallet/Resources/Colors.xcassets/e0dbff.colorset/Contents.json:5]: remove png and use pdf

File: p2p_wallet/Scenes/Main/InvestSolend/Deposit/DepositSolendViewModel.swift [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+//
+
+import Combine
+import FeeRelayerSwift
+import Foundation
+import Resolver
+import SolanaSwift
+import Solend
+
+@MainActor
```
Comment (qwerkly) [inline p2p_wallet/Scenes/Main/InvestSolend/Deposit/DepositSolendViewModel.swift:15]: зачем использовать MainActor?

File: p2p_wallet/Scenes/Main/InvestSolend/InvestSolendView.swift [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+        VStack {
+            // Title
+            HStack {
+                Text(L10n.earnAYield)
+                    .fontWeight(.bold)
+                    .apply(style: .largeTitle)
+                Spacer()
+            }
+            .padding(.top, 20)
+            .padding(.horizontal, 16)
```
Comment (qwerkly) [inline p2p_wallet/Scenes/Main/InvestSolend/InvestSolendView.swift:24]: Don't use this, use navigationTitle instead this. It's a dirty hook

File: p2p_wallet/Scenes/Main/InvestSolend/InvestSolendViewModel.swift [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+import Resolver
+import SolanaSwift
+import Solend
+
+typealias Invest = (asset: SolendConfigAsset, market: SolendMarketInfo?, userDeposit: SolendUserDeposit?)
+
+@MainActor
+class InvestSolendViewModel: ObservableObject {
+    private let service: SolendDataService
+    var subscriptions = Set<AnyCancellable>()
```
Comment (qwerkly) [inline p2p_wallet/Scenes/Main/InvestSolend/InvestSolendViewModel.swift:16]: add private

File: p2p_wallet/Scenes/Main/InvestSolend/InvestView.swift [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+import SwiftUI
+
+struct InvestView: View {
+    var body: some View {
+        NavigationView {
+            InvestSolendView(viewModel: try! .init(mocked: false))
+                .navigationBarHidden(true)
+        }
+    }
+}
```
Comment (qwerkly) [inline p2p_wallet/Scenes/Main/InvestSolend/InvestView.swift:14]: why do you need this view? You embed InvestSolendView inside navigationView inside itself

File: p2p_wallet/Resources/Assets.xcassets/solend-tutorial-2.imageset/Contents.json (hunk 1 above)
Comment (bigearsenal) [reply p2p_wallet/Resources/Assets.xcassets/solend-tutorial-2.imageset/Contents.json:2]: Ok. I'll do it

File: p2p_wallet/Scenes/Main/InvestSolend/Deposit/DepositSolendViewModel.swift (hunk 4 above)
Comment (bigearsenal) [reply p2p_wallet/Scenes/Main/InvestSolend/Deposit/DepositSolendViewModel.swift:15]: For SwiftUI, @MainActor is recommended by Apple
[https://i.stack.imgur.com/Z0E1w.jpg](url)

File: p2p_wallet/Scenes/Main/InvestSolend/InvestSolendView.swift (hunk 5 above)
Comment (bigearsenal) [reply p2p_wallet/Scenes/Main/InvestSolend/InvestSolendView.swift:24]: We've tried but couldn't make navigationTitle to adapt the design.

File: p2p_wallet/Scenes/Main/InvestSolend/InvestSolendViewModel.swift (hunk 6 above)
Comment (bigearsenal) [reply p2p_wallet/Scenes/Main/InvestSolend/InvestSolendViewModel.swift:16]: Ok. I'll do that

File: p2p_wallet/Scenes/Main/InvestSolend/InvestView.swift (hunk 7 above)
Comment (bigearsenal) [reply p2p_wallet/Scenes/Main/InvestSolend/InvestView.swift:14]: @TrGiLong wrote this. Can you explan?

File: p2p_wallet/Scenes/Main/InvestSolend/InvestSolendView.swift (hunk 5 above)
Comment (bigearsenal) [reply p2p_wallet/Scenes/Main/InvestSolend/InvestSolendView.swift:24]: With navigationTitle and largeTitle = true, the title is correct, but the button will be at the top, not inline with the title. So we couldn't use navigationTitle

Comment (bigearsenal) [reply p2p_wallet/Scenes/Main/InvestSolend/InvestSolendView.swift:24]: Design:
image
with navigationTitle and "rightBarButton":
image

Comment (bigearsenal) [reply p2p_wallet/Scenes/Main/InvestSolend/InvestSolendView.swift:24]: There is a solution for this, but it has the risk that application might be rejected by Apple for using private api
https://stackoverflow.com/a/63748032

Comment (bigearsenal) [reply p2p_wallet/Scenes/Main/InvestSolend/InvestSolendView.swift:24]: If you have any ideas, please help us by opening PR. We're appreciated!

**Pull request merged**